src/components/OntologyExplorer/drawForce/hulls.ts

```diff
@@ -94,3 +92,22 @@ const drawHull = (
     context.stroke();
   }
 };
+
+export const getHullNodes = (
+  vertex_id: string,
+  ontology: Ontology,
+  nodes: any,
+): string[] => {
+  /**
+   * Get nodes belonging to a hull
+   */
+  const vertex: any = ontology.get(vertex_id);
+  if (vertex) {
+    const descendants = [...vertex.descendants];
+    return nodes.filter((node: any) => {
+      return descendants.includes(node.id) || node.id === vertex_id; // include self
+    });
+  } else {
+    return [];
+  }
+}
```

Comment:
missing EOL

Please run Prettier with project config before landing these files.

Reply:
Sorry, Bruce. I forgot it's not set up as a commit hook. Just updated all the PRs with a prettier run and will make sure to do so for future PRs.